### PR TITLE
WIP Add docs and improve docs

### PR DESCRIPTION
**Pull request by maarteNNNN**

### CODE REVIEW

File: docs/learn-fees-on-clsk.md [hunk 1]
```diff
@@ -24,3 +24,7 @@ Capitalisk nodes are decentralized and charge fees per processed transaction. Ea
 ```
 
 Any transaction that does not contain enough fees are not being processed. To review fees you can simply run `ldpos fees` in the [`ldpos-commander`](https://github.com/Capitalisk/ldpos-commander).
+
+:::note
+Capitalisk is a supports dynamic fees
```
Comment (jondubois) [inline docs/learn-fees-on-clsk.md]: "Capitalisk supports dynamic fees."

File: docs/learn-unique-features.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ sidebar_label: Unique features
 slug: /unique-features
 ---
 
-- The `ldpos-chain` allows you to change passphrases. A transaction is needed for it to go in effect.
+- The `ldpos-chain` allows you to change passphrases. A transaction is needed for it take effect.
```
Comment (jondubois) [inline docs/learn-unique-features.md]: Should be "for it to take effect" ("to" is missing)

**Review by jondubois — CHANGES_REQUESTED** [review]

File: docs/learn-fees-on-clsk.md (hunk 1 above)
Comment (maarteNNNN) [reply docs/learn-fees-on-clsk.md]: See 6af22f505e71d5d1b388e7b78a9601fbb641cfb1.

File: docs/learn-unique-features.md (hunk 2 above)
Comment (maarteNNNN) [reply docs/learn-unique-features.md]: See 6af22f505e71d5d1b388e7b78a9601fbb641cfb1.

**Pull request merged**